neoassistant/commands.py

```diff
@@ -241,10 +241,22 @@ def execute(self, assistant: Assistant, args):
 
 class ShowBirthdaysCommand(Command):
     def __init__(self):
-        super().__init__("show-birthdays", "Show all birthdays per next 7 days.")
+        super().__init__(
+            "show-birthdays", 
+            "Show all birthdays per the next specified number of days," + 
```

Comment:
Update to `"Show all birthdays per the next specified number of days. Format: show-birthdays <days> (default 7)"`